src/App.tsx

```diff
@@ -2,13 +2,26 @@ import React from 'react';
 
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import getPosts from './api/posts';
+import getComments from './api/comments';
+import getUsers from './api/users';
+import { PostList } from './components/PostList';
+
+const preparedPosts = getPosts.map(post => {
+  const user = getUsers.find(({ id }) => id === post.userId) || null;
+  const comments = getComments.filter(comment => comment.postId === post.id);
+
+  return {
+    ...post,
+    user,
+    comments,
+  };
+});
```

Comment:
```suggestion
const preparedPosts = getPosts.map(post => ({
    ...post,
    user: getUsers.find(({ id }) => id === post.userId) || null,
    comments: getComments.filter(comment => comment.postId === post.id),
  });
```
Трохи коротше. Але для наглядності і так ок